R/tensor.R

```diff
@@ -42,6 +42,13 @@ Tensor <- R7Class(
     print = function(n = 30) {
       cat("torch_tensor\n")
       cpp_torch_tensor_print(self$ptr, n)
+      if (!is.null(self$ptr$grad_fn)) {
+        cat('[ grad_fn = <')
+        cat(self$ptr$grad_fn$print())
+        cat('> ]')
+      } else if (self$ptr$requires_grad && is.null(self$ptr$grad_fn)){
+        cat('[ requires_grad = TRUE ]')
+      }
```

Comment:
Maybe we need a `cat("\n")` after everything to have the same effect as `std::endl`? So the next thing printed to the console starts in its own line?

```suggestion
      }
      cat("\n")
```

Reply:
done